crates/storage_models/src/merchant_connector_account.rs

```diff
@@ -41,6 +44,9 @@ pub struct MerchantConnectorAccountNew {
     pub merchant_connector_id: String,
     pub payment_methods_enabled: Option<Vec<serde_json::Value>>,
     pub metadata: Option<serde_json::Value>,
+    pub connector_label: Option<String>,
```

Comment:
Since you have mentioned that this will be unique identity, why is this field optional?

Reply:
this is taken care in api_models , there can be default values as well